docs/source/developer/mission.rst

```diff
@@ -0,0 +1,76 @@
+Numba Mission Statement
+=======================
+
+Introduction
+------------
+
+This document is the mission statement for the Numba project. It exists to
+provide a clear description of the purposes and goals of the project. This
+document begins by providing some background on use-cases for Numba and
+categories of users. With this context established it continues with a
+description of the overall goals of the Numba project and its future direction.
+
+This is a living document:
+
+=========================== ============
+The first revision date is: March 2022
+The last updated date is:   March 2022
+The next review date is:    TBC
+=========================== ============
+
+Background
+----------
+
+The Numba project provides tools to improve the performance of scientific
+software. It comprises numerous facilities including just-in-time (JIT)
+compilation, extension points for scientific library writers, and a compiler
+toolkit on which new computational acceleration technologies can be explored
+and built.
+
+The user base of Numba includes anyone needing to perform intensive
+computational work, including scientists and practitioners from a wide range of
+disciplines, examples include:
+
+* The most common use case, a user wanting to JIT compile some
+  numerical functions.
+* Users providing JIT accelerated libraries for domain specific use
+  e.g. scientific researchers.
+* Users providing JIT accelerated libraries for use in scientific computing as
+  part of the numerical Python ecosystem.
+* Those writing more advanced JIT accelerated libraries containing their own
+  domain specific data types etc.
+* Compiler extenders wanting to write and explore new compiler use
+  cases/needing a custom compiler.
+* Hardware vendors wanting to extend Numba support to their custom silicon.
+
+Project Goals
+-------------
+
+The primary aims of the Numba project are:
+
+* To make it easier for Python developers to write high performance code.
+* To have a core package with a well defined and limited feature scope that
+  meets the needs of the users without being overly complex.
```

Comment:
sounds a little strange, maybe `... meets the needs of the user base ...` or `... meets the needs of its users ...`

Reply:
Thank you for the suggestions, I opted for `... meets the needs of the user base ...`.